packages/plugins/RSS3/src/SNSAdaptor/pages/DonationsPage.tsx

```diff
@@ -60,19 +60,20 @@ export function DonationPage({ socialAddress, publicKey, userId }: DonationPageP
     )
     const donations = useAvailableCollections(
         kvValue?.proofs ?? EMPTY_LIST,
-        CollectionType.Donations,
         allDonations,
+        CollectionType.Donations,
         userId,
         socialAddress.address,
     )
 
+    const twitterId = userId.toLowerCase()
```

Comment:
This plugin should also work for other SNS websites. Not a proper variable name.